.github/workflows/build-and-release.yml

```diff
@@ -1,6 +1,10 @@
 name: build-and-release
 
-on: push
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
CI currently doesn't run from forks. This enables CI from forks. You can check that ci goes through here: https://github.com/pavelzw/PFERD/actions/runs/8573848476?pr=1